src/main/java/io/vertx/ext/hawkular/impl/VertxMetricsImpl.java

```diff
@@ -73,11 +74,16 @@ public VertxMetricsImpl(Vertx vertx, VertxHawkularOptions options) {
     Context context = vertx.getOrCreateContext();
     sender = new Sender(vertx, options, context);
     scheduler = new Scheduler(vertx, options, context, sender);
-    scheduler.register(httpServerMetricsSupplier);
-    scheduler.register(httpClientMetricsSupplier);
-    scheduler.register(netServerMetricsSupplier);
-    scheduler.register(netClientMetricsSupplier);
-    scheduler.register(datagramSocketMetricsSupplier);
+    if(this.options.isMetricsTypeDisabled(MetricsType.HTTP_SERVER))
```

Comment:
This part has changed in master. The setup is now completed in `#eventBusInitialized` method